packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -620,6 +620,24 @@ const service = new ecs.FargateService(this, 'Service', {
 });
 ```
 
+### Deployment alarms 
+
+Amazon ECS [deployment alarms]
+(https://aws.amazon.com/blogs/containers/automate-rollbacks-for-amazon-ecs-rolling-deployments-with-cloudwatch-alarms/) Given a list of alarms, ECS will conduct the deployment as normal. Then, it will monitor the alarms for a “bake time” determined by the alarm’s sampling and thresholds. If any alarm enters the “ALARM” state during the bake time, the deployment will either be failed (requiring roll-forward) or rolled back. 
```

Comment:
```suggestion
(https://aws.amazon.com/blogs/containers/automate-rollbacks-for-amazon-ecs-rolling-deployments-with-cloudwatch-alarms/) Given a list of alarms, ECS will conduct the deployment as normal. Then, it will monitor the alarms for a “bake time” determined by the alarm’s sampling and thresholds. If any alarm enters the “ALARM” state during the bake time, the deployment will either fail (requiring roll-forward) or be rolled back. 
```

Reply:
changes made in the latest commit.